Adyen/UI/List/CoreListDataSource.swift

```diff
@@ -112,32 +112,26 @@ internal final class CoreListDataSource: NSObject, ListViewControllerDataSource
     
     // MARK: - Item Loading state
     
-    internal func startLoading(for item: ListItem, _ tableView: UITableView) {
-        if let cell = cell(for: item, tableView: tableView) {
-            cell.showsActivityIndicator = true
-        }
+    internal func startLoading(for loadingItem: ListItem, _ tableView: UITableView) {
+        loadingItem.showsActivityIndicator = true
         
         tableView.isUserInteractionEnabled = false
         
-        for case let visibleCell as ListCell in tableView.visibleCells where visibleCell.item != item {
-            visibleCell.isEnabled = false
-        }
-    }
-    
-    private func cell(for item: ListItem, tableView: UITableView) -> ListCell? {
-        for case let cell as ListCell in tableView.visibleCells where cell.item == item {
-            return cell
+        for case let item in sections.flatMap(\.items) where item != loadingItem {
+            item.isEnabled = false
         }
         
-        return nil
+        tableView.reloadData()
```

Comment:
This will reload whole table and cause "flickering", right ?

Reply:
it doesn't flicker :-)